typing_game.py

```diff
@@ -1,103 +1,50 @@
-        self.input.move(155, 200)
-
-        self.word = QtWidgets.QLineEdit(self)
-        self.word.resize(300, 150)
-        self.word.move(105, 30)
-        self.word.setDisabled(True)
-        self.word.setFont(QFont('Times', 40))
-        self.word.setText('Bread')
-
-    # def check_
-
-
-class MainWindow(QMainWindow):
-    def __init__(self):
-        super(MainWindow, self).__init__()
-        self.initUI()
-        self.howto_ = HowTo()
-        self.gamewindow_ = Game()
-
-    def initUI(self):
-        self.setGeometry(300, 300, 400, 300)
-        self.setWindowTitle('Typing Game')
-
-        self.title = QtWidgets.QLabel(self)
-        self.title.move(150, 50)
-        self.title.setFont(QFont('Times', 12))
-        self.title.setText('Typing game')
-
-        self.rules = QtWidgets.QPushButton(self)
-        self.rules.resize(150, 50)
-        self.rules.setText('How to play')
-        self.rules.move(125, 150)
-        self.rules.clicked.connect(self.toggle_howto)
-
-        self.start_game = QtWidgets.QPushButton(self)
-        self.start_game.resize(150, 50)
-        self.start_game.setText('Start Game')
-        self.start_game.move(125, 90)
-        self.start_game.clicked.connect(self.toggle_game)
-
-    def toggle_howto(self):
-        if self.howto_.isVisible():
-            self.howto_.hide()
-        else:
-            self.rules.setText('Close window?')
-            self.howto_.show()
-
-    def toggle_game(self):
-        self.gamewindow_.show()
-
-
-def main():
-    app = QApplication(sys.argv)
-    win = Game()
-    win.show()
-    app.exec_()
-
-
-main()
+from input_box import Inputbox
+from text_widget import Textbox
+from sentences import sentences
+from score_window import Score
+from PyQt5.QtWidgets import QMainWindow
+import time
+import random
+from time import time
+
+
+class Game(QMainWindow):
+    def __init__(self,parent=None):
+        super(Game, self).__init__(parent)
+        self._WPM_info = Score()
+        self.text = random.choice(sentences).split()
+        self.i = 0
+        self.error = 0
+        self.initUI()
+        self.time= None
+
+
+    def initUI(self):
+        self.setGeometry(600, 200, 500, 300)
+        self.setWindowTitle('Typing game')
+
+        self.input_box = Inputbox(self)
+        self.input_box.move(150,225)
+        self.input_box.returnPressed.connect(self.test)
+
+
+        self.text_box = Textbox(self)
+        self.text_box.move(20,30)
+
+    def test(self):
+        if self.typed_words == len(self.text):
+            self.find_WPM(time())
+            self._WPM_info.show()
+        elif self.input_box.text()==self.text_box.text():
+            self.text_box.setText(self.text[self.typed_words])
+            self.typed_words+=1
+            if self.typed_words==1:
+                self.time=time()
+        else:
+            self.error+=1
+
+    def find_WPM(self,end_time):
+        self.time=end_time-self.time
+        self._WPM_info.WPM = 10

```

Comment:
your setting the wrong variable. 
```suggestion
        self._WPM_info._WPM = 10

```